src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (a) => {
+    if (a === undefined) {
```

Comment:
```suggestion
    if (!a) {
```

Reply:
tried to implement your advice but met with next (arg 0 converted to false)  https://prnt.sc/m6zj1hB4LKJa